src/compile/selection/point.ts

```diff
@@ -1,11 +1,49 @@
-import {Stream} from 'vega';
+import {Signal, Stream} from 'vega';
 import {stringValue} from 'vega-util';
-import {SelectionCompiler, TUPLE, unitName} from '.';
+import {SelectionCompiler, TUPLE, isTimerSelection, unitName} from '.';
 import {SELECTION_ID} from '../../selection';
 import {vals} from '../../util';
 import {BRUSH} from './interval';
 import {TUPLE_FIELDS} from './project';
 
+export const CURR = '_curr';
+
+const animationSignals = (selectionName: string, scaleName: string): Signal[] => {
+  return [
+    // timer signals
+    {
+      name: 'anim_clock',
+      init: '0',
+      on: [
+        {
+          events: {type: 'timer', throttle: 16.666666666666668},
+          update:
+            'true ? (anim_clock + (now() - last_tick_at) > max_range_extent ? 0 : anim_clock + (now() - last_tick_at)) : anim_clock'
+        }
+      ]
+    },
+    {
+      name: 'last_tick_at',
+      init: 'now()',
+      on: [{events: [{signal: 'anim_clock'}], update: 'now()'}]
+    },
+    {
+      name: 'eased_anim_clock',
+      // update: 'easeLinear(anim_clock / max_range_extent) * max_range_extent'
+      update: 'anim_clock' // TODO: replace with above once easing functions are implemented in vega-functions
+    },
+
+    // scale signals
+    // TODO(jzong): uncomment commented signals when implementing interpolation
```

Comment:
There still appear to be some commented signals, so I'm not sure what the status of this TODO is?

Reply:
these signals are for a follow-up PR